src/proof/bid_tests.rs

```diff
@@ -237,12 +231,11 @@ mod protocol_tests {
         let latest_consensus_step = 25519u64;
 
         // Append the Bid to the tree.
-        tree.push(bid.into());
+        tree.push(bid.into()).unwrap();
```

Comment:
Same here.

To be clear, if the tests was already using `unwrap()` I wouldn't care much, but this is a kind of "regression".